src/main/java/com/marklogic/mgmt/resource/restapis/RestApiManager.java

```diff
@@ -105,13 +105,24 @@ public boolean deleteRestApi(RestApiDeletionRequest request) {
 				PayloadParser parser = new PayloadParser();
 
 				if (request.isIncludeModules()) {
+					boolean includeModules = true;
 					if (request.isDeleteModulesReplicaForests()) {
-						String modulesDatabase = parser.getPayloadFieldValue(payload, "modules-database");
-						if (databaseManager.exists(modulesDatabase)) {
+						String modulesDatabase = null;
```

Comment:
This prevents an error from occurring when "modules-database" doesn't exist in the server properties - that occurs when the modules database has been set to be the filesystem.